Makefile

```diff
@@ -1,4 +1,5 @@
 GOPATH = $(shell go env GOPATH)
```

Comment:
Let's strive for consistency across the repos in this file. Is this the same as other adapter repos?

Reply:
I'm afraid not, but this is a good way to get GOPATH by dynamic and do not make an effect on the Makefile feature.